source/agora/consensus/EnrollmentManager.d

```diff
@@ -733,7 +733,7 @@ public class EnrollmentManager
     private void collectValidatorStats (Collector collector)
     {
         auto validator_count = validator_set.count();
-        if (!validator_count)
+        if (validator_count)
```

Comment:
Actually I don't think you need the if check at all. There should never be zero active validators in the system.

Reply:
zero can be database error, if you look into the count() function, I think it is better not to report in this case

Reply:
Yeah, we're gonna have to fix all of that later (w.r.t. ValidatorSet itself).

Reply:
Well, I think it's better to have a `0` than no data (no data can happen if the node is offline for example).
So I'd just remove the check as @AndrejMitrovic mentioned.